projects/sample/ReadMe_SampleProject.md

```diff
@@ -20,9 +25,9 @@ There are six main components to the sample project, which are delineated below.
     - <sup>\*</sup>We have special permission from the Princeton Institutional Review Board to publicly post the dicoms on this Github repository. 
         - **IMPORTANT:** DO NOT post the raw dicoms for your experiment on Github!!
 - **`projectMain` script**
-    - This script starts the project, including the webServer.
+    - This script is called when you run the projectInterface. This starts the project on the cloud and the webServer on your browser.
```

Comment:
"This starts the project on the cloud and the webServer on your browser". The webServer runs in the cloud and you browser connects to it.
Maybe:
"This script is called when you run the projectInterface. The projectInterface starts the web server in the cloud and initializes communication with the file server. From a browser window pointed to the web server URL you will have controls to start and stop your project scripts."